tox.ini

```diff
@@ -1,18 +1,19 @@
 [tox]
-envlist = py3-docs
+envlist = docs
 skipsdist = true
+requires =
+  tox-pip-sync
+  tox-pyenv
+  tox-run-command
 
 [testenv]
 skip_install = true
 deps =
-    {docs,checkdocs}: recommonmark
-    {docs,checkdocs}: sphinx-autobuild
-    {docs,checkdocs}: sphinx
-    {docs,checkdocs}: sphinx_rtd_theme
+    {docs,checkdocs}: -r requirements/docs.txt
+    {env:EXTRA_DEPS:}
 passenv =
     HOME
-changedir =
-    {docs,checkdocs}: docs
```

Comment:
Changing dirs isn't necessary, and makes `make requirements` write out local paths to the compiled files.